.github/workflows/ci.yml

```diff
@@ -237,14 +239,16 @@ jobs:
     runs-on: ubuntu-latest
     continue-on-error: true
     steps:
-      - uses: actions/checkout@v4
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
       - id: prepare
         uses: ./.github/actions/prepare
-      - uses: dtolnay/rust-toolchain@beta
+      - uses: dtolnay/rust-toolchain@888c2e1ea69ab0d4330cbf0af1ecc7b68f368cc1 # v1.0.0
         id: toolchain
+        with:
+          toolchain: stable
```

Comment:
a bunch of these were inconsistent in which version was in use. @cypt4 can you make sure  making this consistently use stable toolchain won't break anything. I wasn't sure how to test it.

Reply:
Actually we use it via DEPS, so we don't use any CI artifacts, but compile sources using chromium toolchain

Reply:
I think yes